ArticleTemplates/assets/js/modules/messenger.js

```diff
@@ -0,0 +1,198 @@
+/* global Promise */
```

Comment:
@regiskuckaertz this PR will mean Promise is available https://github.com/guardian/mobile-apps-article-templates/pull/637